javascript/shared/third_parties/crypto_init.yml

```diff
@@ -0,0 +1,19 @@
+type: shared
```

Comment:
I'm not sure if these are strictly "third-party" - perhaps in a "lang" folder is better?

Reply:
Yeah I think `lang` is best for NodeJS stdlib things.